DM-41361: Write characterization report for 26.0.0 Science Pipelines Release (#2)
lsst-dm/DMTR-421: DMTR-421.tex

Between w\_2022\_48 (2022-11-24; the source for pipelines version 25) and w\_2023\_32 (v26 source), most major changes have been in the pipelines ``middleware'' and other pipelines infrastructure, with few changes that would be expected to affect the results of data processing.
The major change that is expected to affect data products is the adoption of a modified form of \href{https://github.com/lsst/gbdes}{gbdes} for determining astrometric solutions in the pipelines. This was accompanied by a switch from using Gaia DR2 to DR3 as the astrometric reference catalog.
However, as shown in this document, the adoption of \texttt{gbdes} did not lead to significant changes in the astrometry metrics as measured on RC2.

> As further adaptation and refinement of \texttt{gbdes} continues these metrics are likely to improve.

Reviewer: ```suggestion
As further adaptation and refinement of \texttt{gbdes} continues we expect to see improvements in these metrics.
```

Reviewer: Very minor subtle point - we expect them to improve based on our understanding but we could be wrong

Some KPMs (e.g., PF1, AF1, AF2) involve thresholds that are different for ``design'', ``minimum'', and ``stretch'' specifications. Metrics in this report are all compared to the ``design'' thresholds. The assessment of these KPMs would be different if evaluated against different thresholds.

%\section{Metric monitoring dashboard}\label{sec:dashboard}
\begin{figure}[!t]
  \centering
  \includegraphics[width=0.6\textwidth, trim=0.0in 0.0in 0.0in 0.0in, clip]{figures/photom_metrics_v26_with_thresholds.png}
  \caption{Photometry metrics PA1 (photometric repeatability) and PF1 (percentage of measurements exceeding the outlier threshold) measured in the $r$-band. The figure shows the values of these metrics as measured in versions 22-26 of the LSST Science Pipelines, compared against the SRD requirements (for both the ``design'' and ``minimum'' thresholds). The measured values of both metrics have improved slightly between the previous release (v25) and the current one (v26), likely due to improvements in the robustness of aperture correction measurements to outliers.}
  \label{fig:phot_metrics}
\end{figure}

> 

Reviewer: Is the y-axis units are missing. Is the "metric value"   mmag (PA1) or % (PF1) value?

Author: Yes, because they have different units (PA1 is mmag, PF1 is %), I just included the units in the legend. If that's too confusing I could make separate plots.
